deployment/media-insights-stack.yaml

```diff
@@ -405,7 +372,7 @@ Resources:
             Principal:
               Service: "states.amazonaws.com"
       Policies:
-        - PolicyName: "TestSFNLambdaAccess"
```

Comment:
For whatever reason, the role we are passing through to the workflow stepfunctions was named weird; gave it a more descriptive name.